graspologic/inference/latent_position_test.py

```diff
@@ -2,19 +2,21 @@
 # Licensed under the MIT License.
 
 import numpy as np
+
 from scipy.linalg import orthogonal_procrustes
 
 from ..align import OrthogonalProcrustes
 from ..embed import AdjacencySpectralEmbed, OmnibusEmbed, select_dimension
 from ..simulations import rdpg
 from ..utils import import_graph, is_symmetric
-from .base import BaseInference
 
 
-class LatentPositionTest(BaseInference):
+def latent_position_test(
+    A1, A2, embedding="ase", n_components=None, test_case="rotation", n_bootstraps=500
```

Comment:
not the biggest deal, but could you change the default number of bootstraps to be the same in ldt and lpt?